plugins/Super_Saw/Makefile

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/make -f
+# Makefile for DISTRHO Plugins #
+# ---------------------------- #
+# Created by falkTX
+#
+
+# --------------------------------------------------------------
+# Project name, used for binaries
+
+NAME = Super_Saw
+
+# --------------------------------------------------------------
+# Files to build
+
+OBJS_DSP = \
+	SuperSaw.cpp.o
+
+#OBJS_UI  = \
+#	InfoExampleUI.cpp.o
+
+# --------------------------------------------------------------
+# Do some magic
+
+include ../Makefile.mk
+
+# --------------------------------------------------------------
+# Enable all possible plugin types
+
+ifeq ($(HAVE_DGL),true)
+ifeq ($(HAVE_JACK),true)
+TARGETS += jack
+endif
+endif
+
+ifeq ($(HAVE_DGL),true)
+TARGETS += lv2_sep
+else
+TARGETS += lv2_dsp
+endif
+
+TARGETS += vst
+
+ADA:
+	gnat make -fPIC blep
```

Comment:
this requires external tools, we need to make sure to make this plugin optional then...
does gnat have a pkg-config file?